### PR TITLE
Create servlet with list of quotes, add button to portfolio to display random quote

### PR DESCRIPTION
**Pull request by efrainmunoz100**
- Creates jsonServlet, which returns a list of quotes
- Adds a button to main portfolio page to display one of the quotes
- Cleans up main page to group all server test buttons under one heading called "Server Tests"
- Updates script to return a random quote from /jsonServlet
- updates pom.xml to add Gson dependency

Screen Shot 2021-03-05 at 10 56 24 AM

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/jsonServlet.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /josnServlet URL, and returns a json object with a list of my favorite quotes. */
+@WebServlet("/jsonServlet")
+public class jsonServlet extends HttpServlet {
+
+List<String> quoteList =  Arrays.asList(
```
Comment (renguyen) [inline portfolio/src/main/java/com/google/sps/servlets/jsonServlet.java:18]: If this is a constant, should you prefix it with public static final? And you can name it in the typical constant convention with all caps and underscores like QUOTE_LIST

File: portfolio/src/main/java/com/google/sps/servlets/jsonServlet.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "I can't believe I put on my good crocs for this.",
+    "Beds are lowkey slept on. ",
+    "If you can't handle me at my worst, imagine how I feel",
+    "I may be stupid \n \n",
+    "Hold on I'm trying to remember the second number of the alphabet",
+    "I'm not like the other girls \n I'm worse", 
+    "Be a hero: Eat the straw so the turtles don't have to :)");
+
+Gson gson = new Gson();
+String json = gson.toJson(quoteList);
```
Comment (renguyen) [inline portfolio/src/main/java/com/google/sps/servlets/jsonServlet.java:30]: This can all be indented

**Review by renguyen — COMMENTED** [review]
Great job! Code is nicely organized and decomposed. I left a few notes about formatting constants and indents before I approve :)

**Comment by efrainmunoz100** [comment]
- new commit syncs new changes with main
- still adds all the features mentioned in the previous commit comment
- also cleans up the portfolio main page, so that all the server test buttons show up under one heading
- also makes changes to jsonServlet, so that constants are labeled public static final, and fixes indentation

Screen Shot 2021-03-08 at 4 11 45 PM

**Review by renguyen — APPROVED** [review]
Thanks for making the changes and posting updated screenshots! :)

**Pull request merged**